### PR TITLE
Model 9 Data Artifact

### PR DESCRIPTION
**Pull request by mattkappel**
### Description
- *Category*:  data artifact
- *JIRA issue*: [MIC-3107](https://jira.ihme.washington.edu/browse/MIC-3107)
- *Research reference*: [Stillbirth probability attributable to hemoglobin](https://vivarium-research.readthedocs.io/en/latest/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/risk_effects/index.html#birth-outcomes)

Adds RR and PAF, along with a refactor of existing RR and PAF code. Black was run on some constants files. I avoided running Black on the loader code because it does undesirable things to chained pandas calls.

### Verification and Testing
Successfully generated artifact using `make_artifact -a`

### CODE REVIEW

File: src/vivarium_gates_iv_iron/constants/data_values.py [hunk 1]
```diff
@@ -64,14 +78,26 @@ class _HemoglobinDistributionParameters(NamedTuple):
...
+RR_STILLBIRTH_PROBABILITY_ATTRIBUTABLE_TO_HEMOGLOBIN = (
+    3.87,
+    1.88,
+    8.06,
+)  # (median, lower, upper) 95% CI
+
 _IFA_EFFECT_SIZE_LOWER = 4.08
 _IFA_EFFECT_SIZE_UPPER = 11.52
-_IFA_EFFECT_SIZE_SD = (_IFA_EFFECT_SIZE_UPPER - _IFA_EFFECT_SIZE_LOWER) / (2 * 1.96)  # 95% CI
+_IFA_EFFECT_SIZE_SD = (_IFA_EFFECT_SIZE_UPPER - _IFA_EFFECT_SIZE_LOWER) / (
```
Comment (rmudambi) [inline src/vivarium_gates_iv_iron/constants/data_values.py:100]: I'm not a huge fan of how black handles mathematical expressions like this :/

**Review by rmudambi — APPROVED** [review]

**Review by albrja — APPROVED** [review]

File: src/vivarium_gates_iv_iron/constants/data_values.py (hunk 1 above)
Comment (mattkappel) [reply src/vivarium_gates_iv_iron/constants/data_values.py:100]: Agree. This is why I held off on the `loader.py` bits. Maybe Black isn't our answer.

Comment (stevebachmeier) [reply src/vivarium_gates_iv_iron/constants/data_values.py:100]: Why not just manually override this expression? (I agree it's ugly - kind of an edge case with a long numerator and short denominator)

Or is it kind of a group "policy" to follow black/pep-8 at all costs?

Comment (mattkappel) [reply src/vivarium_gates_iv_iron/constants/data_values.py:100]: We follow PEP-8, but our use of Black is something that recently was added. I'm not sure if there are things we want to change in our `pyproject.toml`.

Comment (mattkappel) [reply src/vivarium_gates_iv_iron/constants/data_values.py:100]: And I think we've yet to automate it.

**Pull request merged**